gadgets/Makefile

```diff
@@ -8,6 +8,7 @@ GADGETS = \
 	trace_dns \
 	trace_exec \
 	trace_mount \
+	trace_sni \
```

Comment:
nit: please keep them ordered in the list.

Reply:
Hi, Mauricio! Thanks for checking :) Done ✔️